### ISSUE TITLE
When I paste an image in a card that has other images, the addon resizes them all to my "max-width" value (250px)

### ISSUE THREAD

**Issue by alvarodelgadoc** [Arthur-Milchior/anki-resize-image#45]
I have used this addon for months now, and I love the fact it can show a smaller version in the editor. But, since this error appeared it has been a hard to keep editing cards. Before, this didn't happen. I could paste as many images as I wanted and the size of the images were preserved. Now, when I do so, all the images are resized to 250px (too small for reviewing). I have to resize them manually to return to their original size. I thought it could probably be another addon (image resizer), but the problem still happened despite turning it off.

https://user-images.githubusercontent.com/88471301/128293384-ae718933-7784-48f0-ad52-93548476ae3a.mp4

**Comment by Arthur-Milchior** [comment]
Did you try creating a new note with "image resizer" disabled ? It's possible that an add-on leave some problem in an already existing card that still create issues after the add-on is disabled

**Comment by Arthur-Milchior** [comment]
Did you try creating a new note with "image resizer" disabled ? It's possible that an add-on leave some problem in an already existing card that still create issues after the add-on is disabled

**Comment by Arthur-Milchior** [comment]
Did you try creating a new note with "image resizer" disabled ? It's possible that an add-on leave some problem in an already existing card that still create issues after the add-on is disabled

**Comment by alvarodelgadoc** [comment]
Yes, I have done that (disabling image resizer) and still the problem persists. When I turn off this addon (resize image in editor) it fixes, however I cannot see them smaller in the reviewer anymore. When I turn it on again, the problem arises. I cannot paste new images because it messes with the size of the other images.
